libraries/AP_GPS/AP_GPS.h

```diff
@@ -144,6 +144,22 @@ class AP_GPS
         uint32_t last_gps_time_ms;          ///< the system time we got the last GPS timestamp, milliseconds
     };
 
+    struct RTK_State {
+        uint32_t time_last_baseline_ms;     ///< Time since boot of last baseline message received in ms.
+        uint8_t rtk_receiver_id;            ///< Identification of connected RTK receiver.
+        uint16_t wn;                        ///< GPS Week Number of last baseline
+        uint32_t tow;                       ///< GPS Time of Week of last baseline in milliseconds
+        uint8_t rtk_health;                 ///< GPS-specific health report for RTK data.
+        uint8_t rtk_rate;                   ///< Rate of baseline messages being received by GPS, in Hz
+        uint8_t nsats;                      ///< Current number of sats used for RTK calculation.
+        uint8_t baseline_coords_type;       ///< Coordinate system of baseline. 0 == ECEF, 1 == NED
+        int32_t baseline_a_mm;              ///< Current baseline in ECEF x or NED north component in mm.
```

Comment:
Minor style nitpick: Regardless of the mavlink name from a code perspective can we stick with calling these baseline_x_mm (X/Y/Z rather then A/B/C)